docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -31,6 +31,8 @@ To submit a documentation PR from docs.ansible.com with ``Edit on GitHub``:
 #. Be patient while Ansibot, our automated script, adds labels, pings the docs maintainers, and kicks off a CI testing run.
 #. Keep an eye on your PR - the docs team may ask you for changes.
 
+.. _issues_and_prs:
```

Comment:
We should probably save this anchor for a general section on reviewing PRs and issues, rather than the docs-focused section. I believe @gundalow is working on something . . .

Reply:
Got it!  I can remove this anchor, update the above index entry, and re-submit this PR.

Reply:
This edit has been removed.